SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialOutputDevice.cpp

```diff
@@ -12,7 +12,7 @@ FSpatialOutputDevice::FSpatialOutputDevice(USpatialWorkerConnection* InConnectio
 	, PIEIndex(InPIEIndex)
 {
 	const TCHAR* CommandLine = FCommandLine::Get();
-	bLogToSpatial = !FParse::Param(CommandLine, TEXT("NoLogToSpatial"));
+	bLogToSpatial = !FParse::Param(CommandLine, TEXT("NoLogToSpatial")) && !GetDefault<USpatialGDKSettings>()->bDisableLoggingToSpatial;
```

Comment:
Hmmm, I'm concerned about this affecting cloud deployments. If a user disables logging to spatial for their editor environment but then deploys, their logging in the cloud will also be disabled. It depends on our expecations, if a user sets `bDisableLoggingToSpatial` do we want all assemblies from this project to also have logging disabled? Might be worth asking customer needs / wants.